experimental/uefi/loader/src/main.rs

```diff
@@ -27,19 +26,29 @@ use tokio_util::codec::Decoder;
 mod qemu;
 mod server;
 
+#[derive(clap::ArgEnum, Clone, Debug, PartialEq)]
+enum Mode {
+    Uefi,
+    Bios,
+}
```

Comment:
Relatedly, it might be worth changing the folder structure, as the loader & baremetal binary both are nested under the `UEFI` dir. :)

Reply:
Yeah, the directory name is a misnomer by now. I'm inclined to let it be named what it is now, hopefully we pick one direction and move out of experimental in not too distant future.